### PR TITLE
corrections to Connect to Box -- misspellings, missing words, formatting

### PR DESCRIPTION
**Pull request by ReQ6W7rFnaBxyfb**
If this PR is accepted, please label it as "hacktoberfest-accepted" or classify the repository with the "hacktoberfest" topic. Thanks! 
https://hacktoberfest.digitalocean.com/hacktoberfest-update

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/mm-box-connection.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
 
 1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern) and select Box "Get started".
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:25]:
```suggestion
1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern), select the **Get started** box.
```

Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:25]:
```suggestion
1. From the [Migration page in the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=migrationCenter&modern), select Box "Get started".
```

File: migration/mm-box-connection.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
 2. Select **Connect to Box**. 
-3. Mover works in the background to scan Box, and copy data to the Microsoft 365.  Authroize Mover withyour Microsoft credentials to connect your co-admin Box account. Click **Authorize Mover*.
+3. Mover works in the background to scan Box, and copy data to the Microsoft 365.  Authorize Mover with your Microsoft credentials to connect your co-admin Box account. Click **Authorize Mover**.
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:27]:
```suggestion
3. Mover works in the background to scan Box and to copy data to Microsoft 365.  Authorize Mover with your Microsoft credentials to connect your co-admin Box account. Select **Authorize Mover**.
```

File: migration/mm-box-connection.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
-8. use that copied client ID to add the microsoft app.  Go you your Boc account settings to authrozie a new app in your Box account settings and enter the client ID when prompoted.
-9. The Box scree -- click "Authorize new app"
-10. Paste in the client ID you copied from the earlier screen.  Enter the cvlient id for application for which you would like to grant accedss.
-11. An App Authroizetion page appears and lists the autroiztion details of the requst.  Click "Authroize"
+5. Sign in to Box with your co-admin account. This will connect your Box account to Microsoft 365.
+6. Log in to grant access to Box. Click **Authorize.**
+7. Install the Microsoft Box app in your Box account settings. This app will authorize content to be copied from your Box account to Microsoft 365. First copy the app client ID click copy.
+8. Use that copied client ID to add the Microsoft app. Go you your Box account settings to authorize a new app in your Box account settings and enter the client ID when prompted.
+9. In the Box screen -- click "Authorize new app".
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:33]:
```suggestion
9. In the Box screen, select "Authorize new app".
```

File: migration/mm-box-connection.md [hunk 4]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
+10. Paste in the client ID you copied from the earlier screen. Enter the client id for application for which you would like to grant access.
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:34]:
```suggestion
10. Paste in the client ID you copied from the earlier screen. Enter the client id for the application for which you would like to grant access.
```

File: migration/mm-box-connection.md [hunk 5]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
+11. An App Authorization page appears and lists the authorization details of the request. Click "Authorize".
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:35]:
```suggestion
11. An App Authorization page appears and lists the authorization details of the request. Select "Authorize".
```

File: migration/mm-box-connection.md [hunk 6]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
 12. You are now connected to Box. Click Finish to close window.
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:36]:
```suggestion
12. You are now connected to Box. Select **Finish** to close the window.
```

File: migration/mm-box-connection.md [hunk 7]
```diff
@@ -24,15 +24,15 @@ description: "Steps to connect to Box when using Migration Manager in the ShareP
...
 4. Sign in with an account that has [admin permissions](/sharepoint/sharepoint-admin-role) for your organization.  
-5. Sign in to Box with your co-admin account. This will connect your Boc account to Microsoft 365.
-6. Log in to grant access to Box.  Click **Authorize.**
-7. Install the Microsoft Box app in your Box account settings  This app will authorize content to be coied from your Box account to Mcirosoft 365.  First copy the app cliet ID click copy.
-8. use that copied client ID to add the microsoft app.  Go you your Boc account settings to authrozie a new app in your Box account settings and enter the client ID when prompoted.
-9. The Box scree -- click "Authorize new app"
-10. Paste in the client ID you copied from the earlier screen.  Enter the cvlient id for application for which you would like to grant accedss.
-11. An App Authroizetion page appears and lists the autroiztion details of the requst.  Click "Authroize"
+5. Sign in to Box with your co-admin account. This will connect your Box account to Microsoft 365.
+6. Log in to grant access to Box. Click **Authorize.**
```
Comment (JohanFreelancer9) [inline migration/mm-box-connection.md:30]:
```suggestion
6. Log in to grant access to Box. Select **Authorize.**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JoanneHendrickson** [comment]
This was published in error. Deleting.

**Pull request merged**